### PR TITLE
COMMERCE-9384 Drop DDM form dependency 1/4 - backend implementation

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/option/CheckboxCommerceOptionTypeImpl.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+		CPDefinitionOptionRel cpDefinitionOptionRel =
+			_cpDefinitionOptionRelLocalService.fetchCPDefinitionOptionRel(
+				cpDefinitionOptionRelId);
+
+		if (cpDefinitionOptionRel == null) {
+			return false;
+		}
+
+		return cpDefinitionOptionRel.isSkuContributor();
+	}
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/option/CheckboxCommerceOptionTypeImpl.java:98]: why do you reimplement these methods?

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/option/CheckboxMultipleCommerceOptionTypeImpl.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+		CPDefinitionOptionRel cpDefinitionOptionRel =
+			_cpDefinitionOptionRelLocalService.fetchCPDefinitionOptionRel(
+				cpDefinitionOptionRelId);
+
+		if (cpDefinitionOptionRel == null) {
+			return false;
+		}
+
+		return cpDefinitionOptionRel.isSkuContributor();
+	}
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/option/CheckboxMultipleCommerceOptionTypeImpl.java:99]: same question

**Review by andrea-ale-sbarra — COMMENTED** [review]
my comments are more question rather then change requests

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/option/CheckboxMultipleCommerceOptionTypeImpl.java (hunk 2 above)
Comment (alessiorendina) [reply modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/option/CheckboxMultipleCommerceOptionTypeImpl.java:99]: Too much copy-paste

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/option/CheckboxCommerceOptionTypeImpl.java (hunk 1 above)
Comment (alessiorendina) [reply modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/option/CheckboxCommerceOptionTypeImpl.java:98]: Too much copy-paste

**Comment by alessiorendina** [comment]
ci:test:sf

**Pull request closed without merging**